packages/orchestrator-ui-components/src/components/WfoTree/WfoTreeNode.tsx

```diff
@@ -86,6 +87,7 @@ export const WfoTreeNode: FC<WfoTreeNodeProps> = ({
                             onClick={() => toggleSelectedId(item.id)}
                             label={item.label}
                             isActive={selected}
+                            style={{ borderRadius: 6 }}
```

Comment:
Use the `css` prop instead of `style`.
However it seems be used like that in the Tree component already (non blocking for the PR in this case then)

Reply:
ok, i leave it as is.